pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -305,3 +304,13 @@ title: "Professional Development Workshop Survey"
   #thanks{style: 'display: none;'}
     %h1 Thank you for your submission
 
+- if mimeo_url
+  #materials-order{style: 'display: none;'}
+    %p
+      Click
```

Comment:
We should say a little about what mimeo is so people don't think we are just redirecting them to some random website